### PR TITLE
Adds more mob crates for livestock

### PR DESCRIPTION
**Pull request by Peptide90**
## About the PR 
Adds critter crate sprite as well as crates for NPC mob crates for all current animals.
Adds crates for:
**bee, butterfly, cat, chicken, corgi, cow, goat, goose, gorilla, monkeycube, parrot, penguin, snake**

**Screenshots**

livestockcrate
The crate sprite being used. Is from Para but is the same as the TG one (just the door meta is different so used para as fits ours. Cev was a green shipping container so can change but looked disproportional...comments?).
**Changelog**

🆑

- add: Adds Animal / critter mob crates to cargo

### CODE REVIEW

**Comment by Peptide90** [comment]
> 
> 
> Please write a proper changelog

As you wrote that I edited. Will finish this PR tomorrow, ran out of time today.

**Comment by Zumorica** [comment]
> > Please write a proper changelog
> 
> As you wrote that I edited. Will finish this PR tomorrow, ran out of time today.

No worries, take your time!

**Comment by SweptWasTaken** [comment]
Please format RSI correctly

**Comment by Peptide90** [comment]
> 
> 
> Please format RSI correctly

- Corrected RSI issue in latest commit. 
- All tested in game and working at cargo.
- Reduced butterfly mob from 10 to 5 so that it's not so griefable.
- All crates currently cost 1000 points which can be rebalanced in future by whoever should penguin and goose provide for rarer cooking ingredients.
- Added construction graph for the wooden crate (crafting) but currently deconstruct doesn't work so this will need to be looked into. (Historically, crates / lockers needed welded before deconstruction).
- Changed physics mass for mob crates so that they're heavier and also added destructible components so they can be destroyed open.

**Review by PaulRitter — CHANGES_REQUESTED** [review]
please fix conflicts

**Comment by Peptide90** [comment]
> 
> 
> please fix conflicts

Fixed...damn mouse mobs...

File: Resources/Prototypes/Entities/Constructible/Power/cargo_console.yml [hunk 1]
```diff
@@ -40,6 +40,19 @@
       - cargo.funplushies
       - cargo.funartsupplies
       - cargo.funinstruments
+      - livestockbee
```
Comment (SweptWasTaken) [inline Resources/Prototypes/Entities/Constructible/Power/cargo_console.yml:43]: Format these IDs correctly.

**Comment by SweptWasTaken** [comment]
Content part of this PR is good. I disagree with adding them to the cargo console. Remove them and it's mergeable.

File: Resources/Prototypes/Entities/Constructible/Power/cargo_console.yml (hunk 1 above)
Comment (Peptide90) [reply Resources/Prototypes/Entities/Constructible/Power/cargo_console.yml:43]: This is how @Zumorica now wants them. Gone with the cargo prefix. Was going to do the rest at some point.

**Review by PaulRitter — APPROVED** [review]

**Review by PaulRitter — APPROVED** [review]

**Comment by SweptWasTaken** [comment]
Some maintainer can probably fix conflicts really easily for this and then merge it.

**Comment by Peptide90** [comment]
> Some maintainer can probably fix conflicts really easily for this and then merge it.

Will fix Friday unless someone beats me to it.

**Pull request merged**